src/components/SideBar/SideBar.component.tsx

```
@@ -0,0 +1,45 @@
+        <StyledSideBar
+            variant={isMobile ? 'temporary' : 'persistent'}
+            open={isMobile ? open : true}
+        >
+            <Toolbar />
+            <Box flexGrow={1}>
+                <Box overflow="auto">
+                    <NavLinks navItems={NavLinksData.NavItemsTop}></NavLinks>
+                </Box>
+                <Divider />
+                <Box>
+                    <NavLinks navItems={NavLinksData.NavItemsBottom}></NavLinks>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)